### PR TITLE
test-multiple-commenters

### PR DESCRIPTION
**Pull request by ankita-kumari**
Will try and test if multiple approvers can be added.

### CODE REVIEW

**Comment by ketan-joveo** [comment]
howdy ankita

**Comment by ketan-joveo** [comment]
+1

**Comment by ketan-joveo** [comment]
lgtm

File: test_reflow.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 This file is not important. I am just trying to test git-reflow.
+Test multiple comments.
```
Comment (ankita-kumari) [inline test_reflow.txt:2]: Test comment.

**Review by ankita-kumari — COMMENTED** [review]
Testing out comments.

File: test_reflow.txt [hunk 2]
```diff
@@ -1 +1,2 @@
 This file is not important. I am just trying to test git-reflow.
```
Comment (ankita-kumari) [inline test_reflow.txt:1]: This could be changed.

**Review by ankita-kumari — COMMENTED** [review]
test

**Pull request merged**